### ISSUE TITLE
Snomed code do not match with Documentation

### ISSUE THREAD

**Issue by StaticBR** [kassenaerztliche-bundesvereinigung/MIOParser#6]
At the [Dockumentation ](https://mio.kbv.de/pages/viewpage.action?pageId=109842796) the Snomed Code for 
`KBV_PR_MIO_MR_ClinicalImpression_Birth_Examination_Delivery_Information.effectiveDateTime`
is defined as:
![grafik](https://user-images.githubusercontent.com/18309339/161234241-3f3c2349-4fb2-4705-908e-3a7a4b879c7b.png)
But within the Type it is defined as another Snomed Code: 
https://github.com/kassenaerztliche-bundesvereinigung/MIOParser/blob/7db16c85b5de4e0e89874091d07853331446b5ae/src/Definitions/KBV/MR/1.1.0/Profile/MRClinicalImpressionBirthExaminationDeliveryInformation.ts#L128

Which of this definitions is correct?

**Comment by mio42** [comment]
In FHIR wird an der Stelle tatsächlich der Code 412726003 anstatt 1156738004 wie im Informationsmodell verwendet. Das wird in der nächsten Fortschreibung des MIO Mutterpass geprüft und angepasst.

**Comment by StaticBR** [comment]
@mio42  Verstehe ich das richtig, dass ich dann momentan im Mutterpass SSW & Tag der Geburt mit dem SNOMED Code für "Length of gestation at birth"  speichern soll?

**Comment by mio42** [comment]
Genau, momentan muss der Code für "Length of gestation at birth" angegeben werden.
